configuration.sh

```diff
@@ -99,7 +100,11 @@ IRIS_ID="LOCAL_CYTOMINE_IRIS"
 #------------------------------------------------------------------------------
 # PLUGIN: SOFTWARE
 #------------------------------------------------------------------------------
-SOFTWARE_ENABLED=true
+SOFTWARE_DEVELOPMENT=true
+PATH_SOFTWAREROUTER_CONFIG="${HOME}/IdeaProjects/Cytomine-software-router2"
```

Comment:
Do not fill the variable with too specific value. It can be empty, as it is a advanced developer feature.
Also, put an explanation in comment stressing the fact that it's for SR development, because it's not obvious.